tests/console/yast2_clone_system.pm

```diff
@@ -42,24 +42,20 @@ sub run {
     # Check and upload profile for chained tests
     upload_asset $ay_profile_path;
 
-    unless (is_opensuse) {
+    unless (is_opensuse || is_staging) {
         my $devel_repo = get_required_var(is_sle('>=15') ? get_repo_var_name("MODULE_DEVELOPMENT_TOOLS") : 'REPO_SLE_SDK');
         zypper_call "ar -c $utils::OPENQA_FTP_URL/" . $devel_repo . " devel-repo";
     }
 
     zypper_call '--gpg-auto-import-keys ref';
-    zypper_call 'install jing';
+    install_jing();
 
-    my $rc_jing    = script_run "jing $xml_schema_path $ay_profile_path";
-    my $rc_xmllint = script_run "xmllint --noout --relaxng $xml_schema_path $ay_profile_path";
+    my $output_jing = script_output "jing $xml_schema_path $ay_profile_path";
 
-    if (($rc_jing) || ($rc_xmllint)) {
+    if ($output_jing) {
         if (is_sle('<15')) {
             record_soft_failure 'bsc#1103712';
         }
-        else {
-            die "autoinst.xml does not validate for unknown reason";
-        }
```

Comment:
this is intended not die here?

Reply:
yes. this is not needed as the process will fail and report the problem in a previous step and this will never be reached anyway. check http://aquarius.suse.cz/tests/1231